integration/ec2/index.ts

```diff
@@ -21,23 +24,6 @@ class Ec2Stack extends pulumicdk.Stack {
             vpnGateway: true,
             ipAddresses: ec2.IpAddresses.cidr('10.0.0.0/16'),
             natGateways: 1,
-            vpnConnections: {
```

Comment:
This creates a resource, but you can only have a single one of these with the same attributes per region. This test existed to ensure that that resource works, I don't think it is valuable enough to keep testing that it works.